Assets/HoloToolkit/Sharing/Tests/Scripts/ImportExportAnchorManager.cs

```diff
@@ -701,13 +693,14 @@ private void AnchorStoreReady(WorldAnchorStore store)
         /// </summary>
         private void CreateAnchorLocally()
         {
-            if (thisAnchor.isLocated)
+            WorldAnchor anchor = this.EnsureComponent<WorldAnchor>();
```

Comment:
Agreed

Actually, I think that was the original reason why I moved `thisAnchor` into an outer scope.